### PR TITLE
[4.0] Fix sorting list by headers

### PR DESCRIPTION
**Pull request by infograf768**
Pull Request for Issue #21524

### Summary of Changes
Modifications in es6 proposed by @dgrammatiko 

### Testing Instructions
Try ordering any column in back-end

### Result
No more js error. Ordering now works.
Patch testers users, keep away 😸 

@laoneo @wilsonge 
can be merged on review if not tested by people using npm

### CODE REVIEW

File: build/media_src/system/js/searchtools.es6.js [hunk 1]
```diff
@@ -128,10 +128,11 @@
       this.createOrderField();
 
       this.orderCols.forEach((item) => {
-        item.addEventListener('click', () => {
+        item.addEventListener('click', (event) => {
```
[the remark this answers is not in the document]
Comment (infograf768) [reply build/media_src/system/js/searchtools.es6.js:131]: No idea what you mean, dear Houndci...

[the remark this answers is not in the document]
Comment (wilsonge) [reply build/media_src/system/js/searchtools.es6.js:131]: fixed it :)

[the remark this answers is not in the document]
Comment (infograf768) [reply build/media_src/system/js/searchtools.es6.js:131]: Looks like but still does not tell me what had to be done...

[the remark this answers is not in the document]
Comment (infograf768) [reply build/media_src/system/js/searchtools.es6.js:131]: oh, taking off the blank line below

**Comment by infograf768** [comment]
@wilsonge now you can merge. 👍

**Comment by infograf768** [comment]
Please do not merge.
I also find other issues with this file

self.getFilterFields is not a function
and maybe others.

**Comment by infograf768** [comment]
I am closing this because we have too many issues.

**Pull request closed without merging**